### PR TITLE
Issue #sb-forms : checkbox condition added for dependent

### PR DESCRIPTION
**Pull request by pradeepkumarcm96**

### CODE REVIEW

File: projects/common-form-elements/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-    "name": "common-form-elements-v9",
-    "version": "4.6.0",
+    "name": "common-form-elements-web-v9",
```
Comment (kiranharidas187) [inline projects/common-form-elements/package.json:2]: can you revert this change @pradeepkumarcm96

**Review by pallakartheekreddy — APPROVED** [review]
